project/Build.scala

```diff
@@ -2359,7 +2359,12 @@ object ScaladocConfigs {
       .add(VersionsDictionaryUrl("https://scala-lang.org/api/versions.json"))
       .add(DocumentSyntheticTypes(true))
       .add(SnippetCompiler(List(
-        s"${dottyLibRoot}/scala=compile",
+        s"$dottyLibRoot/src/scala=compile",
+        s"$dottyLibRoot/src/scala/quoted=compile",
+        s"$dottyLibRoot/src/scala/compiletime=compile",
+        s"$dottyLibRoot/src/scala/util=compile",
+        s"$dottyLibRoot/src/scala/util/control=compile",
+        "docs/_docs/reference/enums=compile"
```

Comment:
`"docs/_docs/reference/enums=compile"` is independent of `Scala3` or `stableScala3`, so I add it here.